yage-core/src/opengl/texture_loader_web.rs

```diff
@@ -0,0 +1,14 @@
+use crate::{
+    Context,
+    Texture
+};
+
+#[derive(Default)]
+pub struct TextureLoader {
+}
+
+impl TextureLoader {
+    pub fn load(_context: &Context, _texture: &mut Texture, _path: &str) {
+        // [TODO] Implement
+    }
```

Comment:
Not sure yet about the final design, but it will surely involve returning a `Future` for async I/O, so the interface will have to change.

Reply:
Absolutely! That's why I think #24 and #27 should have absolute priority now. We should make the web-example work with the current examples (most importantly the texture-one) and in that process work out a way how to load data files, which works for both the native and the web case.